### ISSUE TITLE
Clean install option

### ISSUE THREAD

**Issue by christoph-bessei** [apioo/fusio#137]
The current fusio installer (```composer create-project fusio/fusio```) just clones the apioo/fusio repository.
It would be nice to have a "clean install"-option, which removes examples and documentation after installation.
This could be done by a composer ```composer create-project fusio/fusio``` script or an own fusio console command e.g. ```php bin/fusio system:removeExamples```. 
The console command is probably better: 
1. It could be interactive, which is not possible with composer scripts: composer/composer#5856
2. The user can decide if and which files should be removed

Examples of (probably) unnecessary files/directories:
* build/
* doc/
* src/
* tests/Api/Todo/
* resources/routes/todo/
* resources/schema/todo/
* resources/migration/*
* resources/routes.yaml (maybe keep the file, but comment the content?)
* resources/migrations.yaml (maybe keep the file, but comment the content?)
* resources/schemas.yaml (maybe keep the file, but comment the content?)
* README.MD
* CHANGELOG.MD

**Comment by k42b3** [comment]
Yes, i think also the console command would be a good solution. We should probably show all files which get deleted through the command, then the user need to confirm the action.

**Comment by k42b3** [comment]
In the latest 1.1.2 release we have a new command `system:clean` which can remove all demo files from the repository. We use this command now also internally but please check whether this works for your case.
